pyo3-build-config/src/import_lib.rs

```diff
@@ -50,6 +51,7 @@ pub(super) fn generate_import_lib(
     ImportLibraryGenerator::new(&arch, &env)
         .version(py_version.map(|v| (v.major, v.minor)))
         .implementation(implementation)
+        .abi_tag(abiflags)
```

Comment:
Renamed to `abiflags()` in the feature branch.

Reply:
Thanks, will update here after the release is pushed!

Reply:
`python3-dll-a` v0.2.11 is up on `crates.io`.

Reply:
Thanks, updated here 👍